Makefile

```diff
@@ -35,4 +37,41 @@ all: $(binary)
 $(BUILD_DIR):
 	mkdir -p $(BUILD_DIR)
 
-.PHONY: all clean install
+ifneq ($(wildcard .git),)
+  tag := $(shell cat VERSION)-$(shell git rev-parse --short HEAD)
+else
+  tag := $(shell cat VERSION)
+endif
+
+# package_name, _version, and _iteration can be overridden by Travis or AppVeyor
+package_base_version ?= $(tag)
+package_iteration ?= "1"
+package_name ?= "pony-stable"
+package_version = $(package_base_version)-$(package_iteration)
+archive = $(package_name)-$(package_version).tar
+package = build/$(package_name)-$(package_version)
+
+# Note: linux only
+define EXPAND_DEPLOY
+deploy: test
+	$(SILENT)bash .bintray.bash debian "$(package_version)" "$(package_name)"
+	$(SILENT)bash .bintray.bash rpm    "$(package_version)" "$(package_name)"
+	$(SILENT)bash .bintray.bash source "$(package_version)" "$(package_name)"
+	$(SILENT)rm -rf build/bin
+	@mkdir -p build/bin
+	@mkdir -p $(package)/usr/bin
+	@mkdir -p $(package)/usr/lib/pony-stable/$(package_version)/bin
+
+	$(SILENT)cp $(BUILD_DIR)/stable $(package)/usr/lib/pony-stable/$(package_version)/bin
+
+	$(SILENT)ln -f -s /usr/lib/pony-stable/$(package_version)/bin/stable $(package)/usr/bin/stable
+	$(SILENT)fpm -s dir -t deb -C $(package) -p build/bin --name $(package_name) --conflicts "pony-stable-master" --conflicts "pony-stable-release" --version $(package_base_version) --iteration "$(package_iteration)" --description "Pony dependency manager" --provides "stable" --provides "pony-stable-release"
+	$(SILENT)fpm -s dir -t rpm -C $(package) -p build/bin --name $(package_name) --conflicts "pony-stable-master" --conflicts "pony-stable-release" --version $(package_base_version) --iteration "$(package_iteration)" --description "Pony dependency manager" --provides "stable" --provides "pony-stable-release"
```

Comment:
Here, particularly: unless these conflicts are packages that exist somewhere, we don't need to list them in the new packages.  So take out these `--conficts`.  And take out the `--provides "pony-stable-release"`, too.